### ISSUE TITLE
julia: command not found

### ISSUE THREAD

**Issue by tkf** [JuliaCI/julia-buildkite-plugin#12]
I'm seeing this failure

> ```
> Julia 1 successfully installed to
> /etc/buildkite-agent/plugins/github-com-JuliaCI-julia-buildkite-plugin-v1/hooks/pre-command: line 144: julia: command not found
> ```
>
> https://buildkite.com/julialang/foldskernelabstractions-dot-jl/builds/32#1bf8859f-f2dd-4a31-90f2-e9712fe10c5c https://github.com/JuliaFolds/FoldsKernelAbstractions.jl/pull/11

from

https://github.com/JuliaCI/julia-buildkite-plugin/blob/2ec318d9e84fd5b6a402d926e7451e8b57b77d6c/hooks/pre-command#L143-L144

The same setting worked before (~ a month ago) https://buildkite.com/julialang/foldskernelabstractions-dot-jl/builds/28#a68e1685-3e7b-4aac-9896-cf5397276d47

**Comment by staticfloat** [comment]
This was my fault; try again now that I've added this fix: https://github.com/JuliaCI/julia-buildkite-plugin/commit/821461fdc59b63e861cff70d9ab55c62589a36a0

**Comment by tkf** [comment]
Thanks a lot for the super quick fix! It works now.

**Comment by maleadt** [comment]
Still seeing this. For example, https://buildkite.com/julialang/cuda-dot-jl/builds/1277#23c774ad-3a6f-42cd-8d19-31f9ebc122c1. If you inspect the plugin checkout, you can see that julia-buildkite-plugin is at 821461f, including the fix. Yet:

```
Julia 1.6-nightly successfully installed to
/etc/buildkite-agent/plugins/github-com-JuliaCI-julia-buildkite-plugin-v1/hooks/pre-command: line 145: julia: command not found
```

This pipeline is installing Julia as `1.6-nightly`, https://github.com/JuliaGPU/CUDA.jl/blob/4e81f3d1f0386324465f10eefd6f74c67a92eb12/.buildkite/pipeline.yml#L145-L165. Strangely, other agents on the same host (cyclops) are using the exact same plugin revision and Julia specification string, and are not failing...

**Comment by maleadt** [comment]
This is definitely not fixed; the other agents that did not error picked up another version of Julia, namely 1.6.0. So I'm guessing the detection of `1.6-nightly` is broken, similarly to how detection of `1` was before https://github.com/JuliaCI/julia-buildkite-plugin/commit/821461fdc59b63e861cff70d9ab55c62589a36a0.

**Comment by DilumAluthge** [comment]
Now that 1.6.0 has been released, are we still uploading 1.6 nightlies to S3?

**Comment by maleadt** [comment]
Yes, from the `release-1.6` branch for upcoming point releases.
